src/suite-design-guide/general-principles.rst

```diff
@@ -104,9 +104,27 @@ files they need are more robust.
 Installing Files At Start-up
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use ``rose suite-run`` *file creation mode* or ``R1``
-install tasks to copy files to the self-contained suite run directory at
-start-up. Install tasks are preferred for time-consuming installations because
+Files can be installed on any remote platforms. As standard, Cylc installs the
```

Comment:
Two possible changes here. The plural is necessary, the change of "As standard,", to "By default" is my preference only.
```suggestion
Files can be installed on any remote platform. By default Cylc installs the
```